config.py

```diff
@@ -30,23 +30,33 @@ class Config:
         "stdout" : "File to output stdout of complier and program",
         "input" : "Stdin for program",
         "error" : "Stderr for program",
+        "link_with_libc" : "Statically link with libc",
+        # "library_path" : "add library path 'dir'",
+        # "link_with" : "link with dynamic or static library 'lib'"
     }
 
     BOOL_OPTIONS: Dict[str, Tuple[List[str], bool]] = {
         "run" : (["-r", "-run"], False),
         "dump" : (["-d", "-dump"], False),
         "dump_tokens" : (["-dt", "-dump_tokens"], False),
         "dump_tc" : (["-dtc", "-dump_tc"], False),
+        "link_with_libc": (["-lc", "--libc"], False),
     }
 
-    REGULAR_OPTIONS: Dict[str, List[str]] = {
+    REGULAR_OPTIONS: Dict[str, Tuple[List[str], bool]] = {
```

Comment:
The type annotation is wrong in master, but the new annotation is wrong as well, it's clearly `Dict[str, Tuple[List[str], str]`
TODO: make this a proper dataclass, maybe?